cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
They aren't hardcoded into images as they can be overwritten with a future COPY. They are just good defaults that the large majority of users should not need to tweak. For those that do we can provide a method to go power mode.